src/main/java/com/zaichick/user/dao/UserDao.java

```diff
@@ -37,4 +41,26 @@ public User getById(String id) {
                 .withConsistentRead(false)
                 .withHashKeyValues(user));
     }
+
+    public List<User> findUserByFnameLnameDOB(String firstName, String lastName, String dateOfBirth) {
+        Calendar cal = Calendar.getInstance();
+        //dont think format is correct
+        SimpleDateFormat sdf = new SimpleDateFormat("EEE MMM dd HH:mm:ss z yyyy", Locale.ENGLISH);
```

Comment:
Date time format should be coming in a ISO8601 format, a basic standards. Also we don't care about time, as it is a birthday